cylc/uiserver/config_defaults.py

```diff
@@ -46,3 +47,10 @@
 c.JupyterHub.cookie_secret_file = f'{RUNTIME_PATH / "cookie_secret"}'
 c.JupyterHub.db_url = f'{RUNTIME_PATH / "jupyterhub.sqlite"}'
 c.ConfigurableHTTPProxy.pid_file = f'{RUNTIME_PATH / "jupyterhub-proxy.pid"}'
+
+# Ensure env variable CYLC_SITE_CONF_PATH is passed to UI Server environment
+c.Spawner.environment = {
+    'CYLC_SITE_CONF_PATH': f'{os.getenv("CYLC_SITE_CONF_PATH")}'}
+
+# remove this before auth goes in...
```

Comment:
I'll remove this before I mark it ready for review. I use it for testing, to ensure I can debug without timeout.